src/src/operationWithArrays/hillel/ArrayToList.java

```diff
@@ -0,0 +1,21 @@
+package operationWithArrays.hillel;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ArrayToList {
+    public static void main(String[] args) {
+        Integer[] ints = {1, 2, 34, 23, 4, 231, 32};
+        var test = toList(ints);
+        System.out.println(test);
+
+    }
+
+    public static <T> List<T> toList(T[] ints) {
```

Comment:
You never know what type of values will be put here, so it's not about `ints`, it's more about `values`. Rename it, please.